Makefile

```diff
@@ -81,12 +81,14 @@ configure-build:
 	mkdir -p $(BUILD_DIR)/usr/tmp $(BUILD_DIR)/usr/lib $(BUILD_DIR)/usr/include
 	mkdir -p $(BUILDLIB_DIR) $(BIN_DIR) $(INCLUDE_DIR)
 
-build-libc: configure-build ## Build libskycoin C client library
+build-libc: configure-build $(BUILDLIB_DIR)/libskycoin.so $(BUILDLIB_DIR)/libskycoin.a ## Build libskycoin C client library
+
+$(BUILDLIB_DIR)/libskycoin.so $(BUILDLIB_DIR)/libskycoin.a: $(LIB_FILES)
```

Comment:
IMO these should be separate make targets . Besides , something I notice now ... `$LIB_FILES` contains `lib/cgo/*.go` files . Nonetheless , if something under `src` changes then library should be recompiled as well .